README.md

```diff
@@ -30,7 +30,7 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Good job @danielamoreno699. But as the previous reviewer said 😄 , please remove all the instructions targeting you. For example:
> "Describe how a new developer could make use of your project."
or
NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

![Annotation 2023-02-1 0](https://user-images.githubusercontent.com/57408419/218880373-8b9726e0-3ca5-4ce0-964e-8152d30cbf12.jpg)


- [ ] Great job @danielamoreno699 😄,  But as the previous reviewer said, your license link doesn't work fine, kindly add an MIT license to your project as this can give your project a more professional look😉. You may delete the `red` section, and create the `MIT.md` file, you may use this as an [example](https://github.com/joel-kalema/MIT/blob/master/MIT.md). Then assign that file to the `MIT` selected in bleu. It'll be something like this  `This project is [MIT](./MIT.md) licensed.` 👍 


![Annotation 2023-02-15 0](https://user-images.githubusercontent.com/57408419/218878027-19898826-b087-4ff6-b9fe-6ae4e6ac658c.jpg)

Reply:
@joel-kalema I checked the README.file and I already deleted all the descriptions. the Prerequisites , setup, install and related topics were not removed because they target other developers.